src/world/Manager/HousingMgr.cpp

```diff
@@ -1632,7 +1632,7 @@ Sapphire::Inventory::HousingItemPtr Sapphire::World::Manager::HousingMgr::getHou
   return Inventory::make_HousingItem( tmpItem->getUId(), tmpItem->getId() );
 }
 
-void Sapphire::World::Manager::HousingMgr::editAppearance( bool isInterior, Sapphire::Entity::Player& player, const Common::LandIdent& landIdent, std::vector< uint16_t > containerList, std::vector< uint8_t> slotList, uint8_t removeFlag )
+void Sapphire::World::Manager::HousingMgr::editAppearance( bool isInterior, Sapphire::Entity::Player& player, Common::LandIdent landIdent, std::vector< uint16_t > containerList, std::vector< uint8_t> slotList, uint8_t removeFlag )
```

Comment:
it should still be const though...

Reply:
It shouldn't matter since it copies the value so the caller cannot be affected?

Reply:
not relevant nor a good excuse, if you're not changing it it should be const. it provides compile time benefits too if you mark an parameteras const as it can perform optimisations that could otherwise be impossible/difficult to deduce with dfa, e.g reusing the value between calls/nested function calls because the compiler is guaranteed that its value couldn't have changed